doc/GeneratorMigration/Library_Inventory.md

```diff
@@ -0,0 +1,453 @@
+# Azure SDK for .NET Libraries Inventory
+
+## Summary
+
+- Total libraries: 391
+- Management Plane (Swagger): 177
+- Management Plane (TSP): 29 (New Generator: 0, Old Generator: 29)
+- Data Plane (Swagger): 114
+- Data Plane (TSP): 34 (New Generator: 2, Old Generator: 32)
```

Comment:
Since now you can specify which generator you want there is no single one "New Generator".  Use the tsp-location.yaml to categorize into which new generator they are opting into by name.

Reply:
Follow the path to the package.json file in the eng folder that is specified in the tsp-location.yaml and use the fully qualified name of the emitter package you find in the dependencies.